### PR TITLE
Refs #30573 -- Rephrased "Of Course" and "Obvious(ly)" in documentation and comments.

### PR DESCRIPTION
**Pull request by adamchainz**
Following #11482, this phrase also implies the reader should already know or have realized what is being said.

### CODE REVIEW

**Comment by adamchainz** [comment]
@rixx thanks for the original change

File: docs/ref/models/fields.txt [hunk 1]
```diff
@@ -1009,7 +1009,7 @@ periodically via e.g. cron).
 .. class:: FilePathField(path=None, match=None, recursive=False, max_length=100, **options)
 
 A :class:`CharField` whose choices are limited to the filenames in a certain
-directory on the filesystem. Has three special arguments, of which the first is
+directory on the filesystem. Has some special arguments, of which the first is
```
Comment (felixxm) [inline docs/ref/models/fields.txt:1012]: This change is not related with the main change. I moved it to a separate commit and added signature fix.

File: docs/ref/models/instances.txt [hunk 2]
```diff
@@ -528,7 +528,7 @@ Updating attributes based on existing fields
 --------------------------------------------
 
 Sometimes you'll need to perform a simple arithmetic task on a field, such
-as incrementing or decrementing the current value. The obvious way to
+as incrementing or decrementing the current value. The simplest way to
```
Comment (felixxm) [inline docs/ref/models/instances.txt:531]: `simplest` was one of words removed in #11482. I rephrased this.

File: docs/faq/usage.txt [hunk 3]
```diff
@@ -2,8 +2,8 @@
 FAQ: Using Django
 =================
 
-Why do I get an error about importing DJANGO_SETTINGS_MODULE?
-=============================================================
+Why do I get an error about importing ``DJANGO_SETTINGS_MODULE``?
+=================================================================
```
[the remark this answers is not in the document]
Comment (adamchainz) [reply docs/faq/usage.txt]: Great

File: docs/ref/models/fields.txt (hunk 1 above)
Comment (adamchainz) [reply docs/ref/models/fields.txt:1012]: Thanks

File: docs/ref/models/instances.txt (hunk 2 above)
Comment (adamchainz) [reply docs/ref/models/instances.txt]: Yeah good point. I think here it was still warranted - it's not saying it's necessarily "simple and obvious" but just that it's the "simplest" option, even if it's not (yet?) obvious or simple for the reader. I think your rephrasing is fine, although it has possibly lost the "signposting" that this first option is not necessarily the best.

Comment (felixxm) [reply docs/ref/models/instances.txt]: Maybe _"A common way of achieving this is ..."_ :thinking:

File: docs/ref/models/instances.txt [hunk 4]
```diff
@@ -528,8 +528,8 @@ Updating attributes based on existing fields
 --------------------------------------------
 
 Sometimes you'll need to perform a simple arithmetic task on a field, such
-as incrementing or decrementing the current value. The obvious way to
-achieve this is to do something like::
+as incrementing or decrementing the current value. You can achieve this by
+doing something like::
```
Comment (adamchainz) [inline docs/ref/models/instances.txt:532]:
```suggestion
as incrementing or decrementing the current value. One way of achieving this
is doing the arithmetic in Python like::
```

I think this captures that it's "a way" of doing it, and signposts that the following paragraphs will explain a "better" way.

I'm not sure about "common" - maybe it's common, maybe it's not, and that doesn't say whether or not it's going to be problematic.

Comment (felixxm) [reply docs/ref/models/instances.txt:532]: Applied.
